src/component/feature/FeatureView/FeatureView.tsx

```diff
@@ -226,7 +226,13 @@ const FeatureView = () => {
                     />
                     <Route
                         path={`/projects/:projectId/features/:featureId/settings`}
-                        component={FeatureSettings}
+                        render={props => (
+                            <FeatureSettings
+                                {...props}
+                                projectId={projectId}
+                                featureId={featureId}
+                            />
+                        )}
```

Comment:
Looks like the other routes take no props and instead extract URL parameters in their own:

```
const { projectId, featureId } = useParams<IFeatureViewParams>();
```